docs/manual/system/settings.de.md

```diff
@@ -525,6 +525,114 @@ Beschreibung.
 | `versionPeriod` | Zeitspanne in Sekunden wie lange ältere Versionen von geänderten Einträgen behalten werden sollen. Standard: `7776000`. |
 
 
+## Umgebungsvariablen
+
+{{< Version "4.9" >}}
+
+In Contao kannst du Umgebungsvariablen verwenden, um das Verhalten der Managed Edition zu beeinflussen. Die Variablen werden in der Datei `.env` definiert und diese Datei muß sich im Hauptverzeichnis der Contao-Installation befinden. Die Nutzung und Bezeichnung der folgenden Variablen sind vorgegeben. Du kannst aber auch beliebige Variablen definieren und dann z. B. in der `config.yml` referenzieren. Falls eine zusätzliche `.env.local` Datei im gleichen Verzeichnis existiert, wird diese automatisch verwendet.
```

Comment:
Environment variables can also be defined on the system level - globally, per user or per process.

Reply:
OK - Than i need help on this.

Reply:
Should we also describe something like in [#5212](https://github.com/contao/contao/pull/5212) here? Or would that be more for the installation/update area. Either way - I would still need suggestions from your side.

Reply:
So ungefähr?

> Umgebungsvariablen sind Variabeln, die auf Betriebssytem-Ebene oder auch pro Benutzer oder sogar Prozess definiert werden können. Du kannst mehr über das Konzept bei Wikipedia (https://de.wikipedia.org/wiki/Umgebungsvariable) erfahren. Der grosse Vorteil bei der Nutzung von Umgebungsvariabeln ist, dass eine Applikation wie Contao so auf den Betrieb innerhalb von Containern vorbereitet ist. Wir werden an dieser Stelle aber den Betrieb in Containern nicht weiter ausführen. Wichtig ist, dass Contao dadurch in diesem Umfeld betrieben werden _kann_. Für den Betrieb ohne die Möglichkeit, selber Umgebungsvariabeln zu setzen, wie etwa auf einem üblichen Shared Hosting, liefert Contao die Möglichkeit mit, Umgebungsvariabeln in einer `.env` Datei zu definieren. Beim Starten interpretiert Contao diese dann und tut so, als wären es echte Umgebungsvariabeln. Dadurch kombiniert Contao das Beste aus beiden Welten: Es ist vorbereitet auf den professionellen Betrieb innerhalb von Containern mit Umgebungsvariabeln, kann aber genauso gut auf einer Shared Hosting-Plattform betrieben werden.

> Solltest du die Variabeln in der `.env` pflegen wollen, muss sich diese im Hauptver.......

Reply:
Thank you @Toflar - Added this now.